CODEOWNERS

```diff
@@ -1,34 +1,39 @@
-*                                               @awslabs/rust-sdk-owners
+*                                                   @awslabs/rust-sdk-owners
 
 # Server
-/codegen-server-test/                           @awslabs/smithy-rs-server
-/codegen-server/                                @awslabs/smithy-rs-server
-/rust-runtime/aws-smithy-http-server/           @awslabs/smithy-rs-server
+/codegen-server-test/                               @awslabs/smithy-rs-server
+/codegen-server/                                    @awslabs/smithy-rs-server
+/rust-runtime/aws-smithy-http-server/               @awslabs/smithy-rs-server
 
 # Python Server
-/codegen-server-test/python/                    @awslabs/smithy-rs-python-server @awslabs/smithy-rs-server
-/codegen-server/python/                         @awslabs/smithy-rs-python-server @awslabs/smithy-rs-server
-/rust-runtime/aws-smithy-http-server-python/    @awslabs/smithy-rs-python-server @awslabs/smithy-rs-server
+/codegen-server-test/python/                        @awslabs/smithy-rs-python-server @awslabs/smithy-rs-server
+/codegen-server/python/                             @awslabs/smithy-rs-python-server @awslabs/smithy-rs-server
+/rust-runtime/aws-smithy-http-server-python/        @awslabs/smithy-rs-python-server @awslabs/smithy-rs-server
+
+# Typescript Server
+/codegen-server-test/typescript/                    @awslabs/smithy-rs-typescript-server @awslabs/smithy-rs-server
+/codegen-server/typescript/                         @awslabs/smithy-rs-typescript-server @awslabs/smithy-rs-server
+/rust-runtime/aws-smithy-http-server-typescript/    @awslabs/smithy-rs-typescript-server @awslabs/smithy-rs-server
```

Comment:
Hum, the group exists: https://github.com/orgs/awslabs/teams/smithy-rs-typescript-server/members

Do I have to do anything else here? 😕

Reply:
I think you need to add smithy-rs repo to the team: https://github.com/orgs/awslabs/teams/smithy-rs-typescript-server/repositories